WooCommerce/Classes/ViewModels/CardPresentPayments/CardPresentModalSuccess.swift

```diff
@@ -0,0 +1,77 @@
+import UIKit
+
+/// Modal presented when the payment has been collected successfully
+final class CardPresentModalSuccess: CardPresentPaymentsModalViewModel {
+
+    /// Amount charged
+    private let amount: String
+
+    /// Closure to execute when primary button is tapped
+    private let printReceiptAction: () -> Void
+
+
+    /// Closure to execute when secondary button is tapped
+    private let emailReceiptAction: () -> Void
+
```

Comment:
Did you forget to hook up noThanks?

Reply:
Yep, that would be the auxiliary button, I guess. Will add in #4069